src/jit/inlinepolicy.cpp

```diff
@@ -26,7 +26,19 @@
 
 InlinePolicy* InlinePolicy::GetPolicy(Compiler* compiler, bool isPrejitRoot)
 {
-    // For now, always create a Legacy policy.
+
+#ifdef DEBUG
+
+    // If JitInlineRandomSeed is set, use the RandomPolicy.
+    unsigned seed = JitConfig.JitInlineRandomSeed();
```

Comment:
Note that we already have STRESS_INLINE mode that increases the multiplies and the threshold IL size of the method to be considered for in-lining.  But only legacy policy is supporting STRESS_INLINE.  May be we should rename it as STRESS_INLINE_LEGACY since random policy is not affected by it.